snuba/reader.py

```diff
@@ -36,11 +37,28 @@ def execute(
         raise NotImplementedError
 
 
-class NativeDriverReader(Reader):
+DATE_TYPE_RE = re.compile(r"(Nullable\()?Date\b")
+DATETIME_TYPE_RE = re.compile(r"(Nullable\()?DateTime\b")
+
+
+def transform_date_columns(result: Result) -> Result:
```

Comment:
How do you feel about having some tests for this function?

Reply:
There should be general coverage via the API tests, but a unit test wouldn't be a bad thing either. At the same time, this is all pretty likely going to go through another round of changes in the near future so I'm hesitant to invest too much time in it.

Reply:
Fair :+1:

Reply:
There will probably be a change soon to change the type of `Result`, I promise to add them at that point (and fix [the bug with non-scalar values noted here](https://github.com/getsentry/snuba/pull/414#issuecomment-521361553).)

I'm putting this in writing so you can hold me to it.  🙂